tests/TestCase/View/ViewBuilderTest.php

```diff
@@ -66,9 +66,16 @@ public function arrayPropertyProvider()
     public function testStringProperties($property, $value)
     {
         $builder = new ViewBuilder();
-        $this->assertNull($builder->{$property}(), 'Default value should be null');
-        $this->assertSame($builder, $builder->{$property}($value), 'Setter returns this');
-        $this->assertSame($value, $builder->{$property}(), 'Getter gets value.');
+        $setter = 'set' . ucfirst($property);
+        $getter = 'get' . ucfirst($property);
+        if (is_bool($value)) {
+            //for boolean values the setter and getter look different
+            $setter = 'enable' . ucfirst($property);
+            $getter = 'is' . ucfirst($property) . 'Enabled';
+        }
```

Comment:
Boolean properties should probably be separate test cases.

Reply:
makes sense - moved `autoLayout` to it's own test.